### PR TITLE
refactor: use native `{Promise}` instead of `bluebird`

### PR DESCRIPTION
**Pull request by evilebottnawi**
Node which we support have native implementation `Promise`.

### CODE REVIEW

File: src/processPattern.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import Promise from 'bluebird';
+import globby from 'globby';
+import createThrottle from 'async-throttle';
```
Comment (michael-ciniawsky) [inline src/processPattern.js]: [p-all](https://github.com/sindresorhus/p-all)
[p-limit](https://github.com/sindresorhus/p-limit)
[p-throttle](https://github.com/sindresorhus/p-throttle)

[async-throttle](https://github.com/zeit/async-throttle) is **deprecated**

File: src/processPattern.js [hunk 2]
```diff
@@ -16,9 +15,11 @@ export default function processPattern(globalRef, pattern) {
         return Promise.resolve();
     }
 
+    const throttle = createThrottle(concurrency || 100);
+
     info(`begin globbing '${pattern.absoluteFrom}' with a context of '${pattern.context}'`);
-    return globAsync(pattern.absoluteFrom, globArgs)
-    .map((fileFrom) => {
+    return globby(pattern.absoluteFrom, globArgs)
+    .then((filePaths) => Promise.all(filePaths.map((fileFrom) => throttle(() => {
```
Comment (michael-ciniawsky) [inline src/processPattern.js]: Please indent 2 by spaces, `filePaths` => `files || paths`, `fileFrom` => `file || from`

File: src/processPattern.js (hunk 1 above)
Comment (evilebottnawi) [reply src/processPattern.js]: @michael-ciniawsky archived, not deprecated :+1:

Comment (d3viant0ne) [reply src/processPattern.js]: None the less I agree with @michael-ciniawsky , i'd prefer using something that is actively maintained

**Comment by evilebottnawi** [comment]
@michael-ciniawsky @d3viant0ne friendly ping

**Pull request merged**